resources/idlharness.js

```diff
@@ -943,6 +941,17 @@ IdlArray.prototype.collapse_partials = function()
     this.partials = [];
 }
 
+IdlArray.prototype.are_different_members = function(m1, m2) {
+    if (m1.name !== m2.name) {
+        return true;
+    } else if (m1.type === 'operation' && m2.type === 'operation'
+        && m1.arguments.length !== m2.arguments.length) {
+        // Method overload.
```

Comment:
This relaxes a potentially incorrect constraint, but there is actually one case where we shouldn't allow overloads either - and that's across separate interface/partial interface/mixin definitions:
https://heycam.github.io/webidl/#idl-overloading

Reply:
Rather than this, can the `arguments` arrays be deeply compared instead?

Reply:
```suggestion
        // Method overload. TODO: Deep comparison of arguments.
```